src/components/CalendarMonth.vue

```diff
@@ -42,9 +42,9 @@ export default {
       type: Number,
       default: 0,
     },
-    firstDayOfWeek: {
+    weekStartDay: {
```

Comment:
I decided to change the name of the property so `firstDayOfWeek` is the external property and `weekStartDay` is the internal one (already validated by parent).